torch/onnx/symbolic.py

```diff
@@ -1710,3 +1710,24 @@ def argmin(g, input, dim, keepdim):
         dim = _parse_arg(dim, 'i')
         keepdim = _parse_arg(keepdim, 'i')
         return g.op('ArgMin', input, axis_i=dim, keepdims_i=keepdim)
+
+
+@parse_args('v')
+def floor(g, self):
+    return g.op('Floor', self)
+
+
+@parse_args('v')
+def ceil(g, self):
+    return g.op('Ceil', self)
+
+
+@parse_args('v')
+def log2(g, self):
+    _ln2 = 0.693147180559945309
+    return g.op('Div', log(g, self), g.op('Constant', value_t=torch.Tensor([_ln2])))
+
+
+@parse_args('v')
```

Comment:
can we also add a test case for this one?

Reply:
done.